Cargo.toml

```diff
@@ -54,7 +54,7 @@ getrandom = { version = "0.2", default-features = false }
 hex = "0.4.3"
 heck = "0.4.1"
 impl-serde = { version = "0.4.0" }
-jsonrpsee = { version = "0.16" }
+jsonrpsee = { version = "0.20" }
```

Comment:
Wierd; I updated jsonrpsee to 0.19 not long ago; I wonder where those changes went!

Reply:
(Maybe I got distracted and never actually merged those changes, whoops!)